Dispatchr/app/containers/App.js

```diff
@@ -20,16 +19,16 @@ const getSceneStyle = (props, computedProps) => {
   return style;
 };
 
-export default class App extends Component {
+class App extends Component {
   render() {
     return (
-      <Provider store={configureStore()}>
```

Comment:
You have to initialize the Provider in two places instead of one this way though